android/build.gradle

```diff
@@ -1,506 +1,5 @@
-buildscript {
```

Comment:
Sad to lose the history of this file.

Reply:
agree, I can probably make the move in 2 steps. 1 commit to move the file to /android/app, and another to add the new version to /android. This way the file is marked as moved and we keep the history.